tests/README.md

```diff
@@ -86,6 +86,16 @@ WITH_VALGRIND=1 tmt run -v
 
 If `valgrind` detects a memory leak in a test, the test will fail, and the logs will be found in the test `data` directory.
 
+## Running integration tests with code coverage
+
+To run integration tests with `coverage`, set `WITH_COVERAGE` environment variable as follows:
+
+```shell
+WITH_COVERAGE=1 tmt run -v
+```
+
+the  coverage will get all the gcda and gcno files to the `bluechi-coverage` dir under `data` dir.
+
```

Comment:
Hmm, I'd probably mention coverage options in README only after the last step, where we will have the coverage report generated